geoip2/records.py

```diff
@@ -841,6 +851,7 @@ def __init__(
         autonomous_system_organization: Optional[str] = None,
         connection_type: Optional[str] = None,
         domain: Optional[str] = None,
+        is_anycast: bool = False,
```

Comment:
This and on line 880 are still out of order!

Reply:
Oh, sorry about that.